apk/testdata/apkTest.dockerfile

```diff
@@ -0,0 +1,25 @@
+# run using:
+#docker build -t dantest -f ./apk/testdata/apkTest.dockerfile . && docker run -it dantest
+
+#FROM alpine
+FROM golang:1.13-alpine
+
+WORKDIR /go/src/app
+COPY . .
```

Comment:
move the COPY further down, right before running the compile/test, that will be much quicker during repeated testing as it allows to cache more of the layers.

Reply:
> ...move the COPY...

Cool! Done in b190238

BTW, I don't actually use this file yet in an automated way. I was manually stopping the unit test that created an .apk, copying the .apk somewhere to save it, then spinning up the docker image and trying to install that .apk.